libs/langchain-mongodb/langchain_mongodb/graphrag/graph.py

```diff
@@ -0,0 +1,482 @@
+from __future__ import annotations
+
+import json
+import logging
+from copy import deepcopy
+from importlib.metadata import version
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Union
+
+from langchain_core.documents import Document
+from langchain_core.language_models.chat_models import BaseChatModel
+from langchain_core.messages import AIMessage
+from langchain_core.prompts.chat import ChatPromptTemplate, SystemMessagePromptTemplate
+from langchain_core.runnables import RunnableSequence
+from pymongo import MongoClient, UpdateOne
+from pymongo.collection import Collection
+from pymongo.driver_info import DriverInfo
+from pymongo.results import BulkWriteResult
+
+from langchain_mongodb.graphrag import example_templates, prompts
+
+from .prompts import rag_prompt
```

Comment:
Can we please make all of these constants UPPERCASE?  That is the convention used by the [sql_toolkit](https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/agent_toolkits/sql/prompt.py) and I find it easier to tell what is a constant and what is a user input.